test/bdd/fixtures/agent-rest/docker-compose.yml

```diff
@@ -12,8 +12,8 @@ services:
     image: ${AGENT_REST_IMAGE}:${AGENT_REST_IMAGE_TAG}
     environment:
       - ARIESD_API_HOST=${ALICE_HOST}:${ALICE_API_PORT}
-      - ARIESD_INBOUND_HOST=${ALICE_HOST}:${ALICE_INBOUND_PORT}
-      - ARIESD_INBOUND_HOST_EXTERNAL=http://alice.aries.example.com:${ALICE_INBOUND_PORT}
+      - ARIESD_INBOUND_HOST=http@${ALICE_HOST}:${ALICE_INBOUND_PORT}
```

Comment:
minor: schema can be moved to `.env` file

Reply:
[Moved](https://github.com/hyperledger/aries-framework-go/compare/e5a6403471837f561ea71f34e3c06e00a9f6d907..1f9c8c18c2255f071b6aa295fbde2d85e23aa7d2) to .env file.